JavaWebService.iml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module version="4">
```

Comment:
You probably don't want to commit this file

Reply:
done